conda/recipes/rmm/meta.yaml

```diff
@@ -40,27 +42,32 @@ requirements:
     - {{ compiler('cuda') }} {{ cuda_version }}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
+    - python
     - cudatoolkit ={{ cuda_version }}
-    - cython >=0.29,<0.30
     - librmm ={{ version }}
-    - python
-    - scikit-build >=0.13.1
-    - setuptools >=61.0.0
-    - tomli  # [py<311]
+    {% for r in data.get("build-system", {}).get("requires", []) %}
+    {% if r.split()[0] not in ["cmake", "ninja"] %}
```

Comment:
Was going to try to ignore `cmake` and `ninja` here, but [`scikit-build` pulls them in anyways]( https://github.com/conda-forge/scikit-build-feedstock/blob/f22722fca3979fc2587cbbfb51592a26a391c105/recipe/meta.yaml#L27-L29 )